src/components/util/CodelabCard/index.js

```diff
@@ -25,48 +25,107 @@ const CardComponent = ({
     setLogoPath(org);
   }, [org]);
 
+ const getDates = (str) => {
```

Comment:
Overall, the feature seems to be important, but I'm not quite sure about the approach which is used here. I feel like we can take the difference and use the diff for further processing.

let diff = Math.floor(date1.getTime() - date2.getTime());
let day = 1000 * 60 * 60 * 24;

let days = Math.abs(Math.floor(diff/day))
let months = Math.abs(Math.floor(days/30));
days = days - (months) * 30;
let years = Math.abs(Math.floor(months/12));
months = months - (years) * 12

This might not return the absolute diff, but this will give the basic idea. This is just a suggestion and please consider others' opinion on this approach.

Reply:
And the function name does not make any sense.

Reply:
Also, might I suggest a package to take a look at. @Janvi-Thakkar 

https://day.js.org/en/

Reply:
@shehand Thank you for suggesting another approach.

Reply:
@Niweera I have heard about the moment.js but I was quite unsure about adding the dependency without asking the maintainers or members. But this suggested library is lightweight at the moment. I would like to get your permission to use it.

Reply:
@Janvi-Thakkar, you should not use moment.js it is really outdated. But I suggest that you should use day.js since I use it regularly. But I would advise you to read the day.js documentation thoroughly before using it anywhere. (as a rule of thumb, always read the documentation of a package you intend to use.)

Reply:
@Niweera I  read docs and some articles for comparing both. I added dayjs library. @Niweera @shehand please review it.

Reply:
will do.

Reply:
Thank You!